### PR TITLE
Update to IIDM version 1.13

### PR DESCRIPTION
**Pull request by colinepiloquet**
**Please check if the PR fulfills these requirements**

- [X] The commit message follows our guidelines

**Does this PR already have an issue describing the problem?**
No

**What kind of change does this PR introduce?**
Feature

**What is the current behavior?**
The (X)IIDM version is 1.12.

**What is the new behavior (if this is a feature change)?**
The IIDM version is **1.13**.

**Does this PR introduce a breaking change or deprecate an API?**
No

### CODE REVIEW

File: iidm/iidm-serde/src/test/resources/V1_13/eurostag-tutorial1-lf-extensions.bin
Comment (olperr1) [inline iidm/iidm-serde/src/test/resources/V1_13/eurostag-tutorial1-lf-extensions.bin:1]: @flo-dup Shouldn't this file (and its counterparts in the other versioned directories) also be removed?

File: iidm/iidm-serde/src/test/resources/V1_13/xsd/terminalMock.xsd [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema version="1.6"
```
Comment (olperr1) [inline iidm/iidm-serde/src/test/resources/V1_13/xsd/terminalMock.xsd:2]: This version number should also be changed:
```suggestion
<xs:schema version="1.13"
```

File: iidm/iidm-serde/src/test/resources/xsd/terminalMockNoSerDe.xsd [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xs:schema version="1.6"
```
Comment (olperr1) [inline iidm/iidm-serde/src/test/resources/xsd/terminalMockNoSerDe.xsd:2]: This version number should also be changed:
```suggestion
<xs:schema version="1.13"
```

File: iidm/iidm-serde/src/test/resources/xsd/terminalMockNoSerDe.xsd [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
            xmlns:xs="http://www.w3.org/2001/XMLSchema"
-           xmlns:iidm="http://www.powsybl.org/schema/iidm/1_12"
+           xmlns:iidm="http://www.powsybl.org/schema/iidm/1_13"
            targetNamespace="http://www.powsybl.org/schema/iidm/ext/terminal_mock_no_serde/1_12"
```
Comment (olperr1) [inline iidm/iidm-serde/src/test/resources/xsd/terminalMockNoSerDe.xsd:5]: This version number should also be changed:
```suggestion
           targetNamespace="http://www.powsybl.org/schema/iidm/ext/terminal_mock_no_serde/1_13"
```

File: iidm/iidm-serde/src/test/resources/xsd/terminalMockNoSerDe.xsd [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
            elementFormDefault="qualified">
-    <xs:import namespace="http://www.powsybl.org/schema/iidm/1_12" schemaLocation="iidm_V1_12.xsd"/>
+    <xs:import namespace="http://www.powsybl.org/schema/iidm/1_13" schemaLocation="iidm_V1_12.xsd"/>
```
Comment (olperr1) [inline iidm/iidm-serde/src/test/resources/xsd/terminalMockNoSerDe.xsd:7]: This version number should also be changed:
```suggestion
    <xs:import namespace="http://www.powsybl.org/schema/iidm/1_13" schemaLocation="iidm_V1_13.xsd"/>
```

**Review by olperr1 — CHANGES_REQUESTED** [review]

File: iidm/iidm-serde/src/test/resources/V1_13/eurostag-tutorial1-lf-extensions.bin
Comment (colinepiloquet) [reply iidm/iidm-serde/src/test/resources/V1_13/eurostag-tutorial1-lf-extensions.bin:1]: I have removed the one in 1.13 and 1.12, I think it was the only two.

**Review by olperr1 — APPROVED** [review]

**Pull request merged**